docs/actions/mongo.md

```diff
@@ -0,0 +1,177 @@
+!!! important "Target Configuration"
+    For all actions, the target should have a property `databaseName`
+
+```json title="Mongo target example"
+{
+    "name": "mongo_target",
+    "url": "mongo://my.mongo.base:27017",
+    "properties": {
+        "databaseName": "myDatabaseName",
+        "username": "myUsername", // (1)
+        "password": "myPassword" // (2)
+    }
+}
+```
+
+1. Valid properties are `username` or `user`. Set this for basic authentication
+2. Valid properties are `userPassword` or `password`. Set this for basic authentication
+
+# Count
+
+=== "Inputs"
+
+    | Required | Name         | Type    |  Default   |
+    |:--------:|:-------------|:--------|:----------:|
+    |    *     | `target`     | String  |            |
+    |    *     | `collection` | String  |            |
+    |    *     | `query`      | String  |            |
+
+=== "Outputs"
+
+    |    Name | Type   |
+    |--------:|:-------|
+    | `count` | long   |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+MongoCountTask(
+    target = "mongo_target",
+    collection = "myCollection",
+    query = "my query"
```

Comment:
for all queries, you could set a real mongodb query, for example :
collection = "ghibli_movies",
query = "{ name: { $in: [ 'Pom Poko', 'Spirited Away' ] } }"